android/src/com/mapswithme/util/Utils.java

```diff
@@ -246,6 +246,29 @@ public static void openUrl(@NonNull Context context, @Nullable String url)
     }
   }
 
+  public static boolean openUri(@NonNull Context context, @NonNull Uri uri)
+  {
+    final Intent intent = new Intent(Intent.ACTION_VIEW);
+    intent.setData(uri);
+    try
+    {
+      context.startActivity(intent);
+      return true;
+    }
+    catch (ActivityNotFoundException e)
+    {
+      CrashlyticsUtils.logException(e);
+      return false;
+    }
+    catch (AndroidRuntimeException e)
+    {
+      CrashlyticsUtils.logException(e);
+      intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+      context.startActivity(intent);
```

Comment:
Is it possible that exception will be thrown again?

Reply:
I guess no. I've done the same handing as it was in openUrl method in this file. And according to the comment to that commit @yoksnod  fixed the problem with starting activity in such way. Also, we don't see the crashes about this in Fabric. So, I can assume we don't have problem with it right now.